### PR TITLE
Fix tests by using grpc-status instead of HTTP status

### PR DESCRIPTION
**Pull request by marcospereira**
Follow up to https://github.com/playframework/play-grpc/pull/45.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -22,7 +22,7 @@ ThisBuild / javacOptions ++= List(
   "-Xlint:deprecation",
 )
 
-resolvers += Resolver.bintrayRepo("akka", "maven")
+ThisBuild / resolvers += Resolver.bintrayRepo("akka", "maven")
```
Comment (marcospereira) [inline build.sbt]: @raboof this is why it was not resolving the dependencies.

File: play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java [hunk 2]
```diff
@@ -61,15 +61,15 @@ public void returns404OnNonGrpcRequest() throws Exception {
...
-    assertEquals(200, rsp.getStatus()); // Maybe should be a 426, see #396
+    assertEquals(500, rsp.getStatus()); // Maybe should be a 426, see #396
   }
 
   @Test
-  public void returns200OnEmptyRequestToAGrpcMethod() throws Exception {
+  public void returns500OnEmptyRequestToAGrpcMethod() throws Exception {
     final WSResponse rsp = wsGet("/" + GreeterService.name + "/SayHello");
-    assertEquals(200, rsp.getStatus()); // Maybe should be a 426, see #396
+    assertEquals(500, rsp.getStatus()); // Maybe should be a 426, see #396
```
Comment (marcospereira) [inline play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java]: Out of the scope here, but it is weird that Scala and Java versions are behaving differently here. Further discussion at https://github.com/akka/akka-grpc/issues/396.

File: build.sbt (hunk 1 above)
Comment (raboof) [reply build.sbt]: Aah, I must've already had them in my ~/.ivy2 so I didn't notice

File: play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java (hunk 2 above)
Comment (raboof) [reply play-testkit/src/test/java/play/grpc/testkit/PlayJavaFunctionalTest.java]: Agreed ;)

**Pull request merged**